view.php

```diff
@@ -97,28 +94,29 @@
 		$c11->colspan = 9;
 		$c11->style   = "padding-bottom:1em;";
 		$c11->text    = html_writer::tag("span",
-		                                 get_string("form_header_question", "block_coursefeedback", $question->id) . ": ",
-		                                 array("style" => "font-weight:bold;"))
-		              . format($question->question);
+		                                 get_string("form_header_question", "block_coursefeedback", $question->questionid) . ": ",
+		                                 array("style" => "font-weight: bold; font-size: 1.5rem"))
+		              . html_writer::tag("span", $question->question, array("style" => "font-size: 1.5rem"));
```

Comment:
Muss hier nicht escaped werden?

Reply:
Fixed in: d90b47827251ad5085e9fc6d9f6246b9d81c6996

Muss ich eigtl den editorinput (den text) irgendwie filtern bevor ich den in der DB soeichere?
Ich erlaube ja keine files:
https://docs.moodle.org/dev/lib/formslib.php_Form_Definition#editor

Reply:
Nein, soweit ich weiß kannst du den einfach abspeichern.